### DISCUSSION TITLE
Saving credentials for mox transfer / login

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1097] [Q&A] [answered]
Is there a way to not have to log in everytime a file transfer is made?

**Comment by kubu4** [comment, marked as answer]
Yes, if you are on Mox and transfer out then no credentials needed for each transfer.

**Comment by kubu4** [comment]
Otherwise, I don't think so. I know they've disable the functionality of using ssh keys (I've emailed them about this), which eliminates the ability to login without a password.
